app/buyers/views/buyers.py

```diff
@@ -270,8 +279,226 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
+                    else:
+                        row.append(brief_response.get(key))
+                csv_rows.append(row)
+
+        return Response(
+            csv_generator.iter_csv(csv_rows),
+            mimetype='text/csv',
+            headers={
+                "Content-Disposition": (
+                    "attachment;filename=responses-to-requirements-{}.csv"
+                ).format(brief['id']),
+                "Content-Type": "text/csv; header=present"
+            }
+        ), 200
+
+    def generate_ods(self, brief, responses):
+        doc = SpreadSheet()
+
+        doc.add_font(FontFace(name="Arial", fontfamily="Arial"))
+
+        doc.add_style("default", "table-cell", (
+            TableCellProperties(wrapoption="wrap", verticalalign="top"),
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="11pt"),
+        ))
+
+        doc.add_style("header", "table-cell", (
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="21pt",
+                           fontweight="bold"),
+        ))
+
+        doc.add_style("title", "table-cell", (
+            TableCellProperties(wrapoption="wrap", verticalalign="top"),
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="16pt",
+                           fontweight="bold"),
+        ))
+
+        doc.add_style("subtitle", "table-cell", (
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="16pt"),
+        ))
+
+        doc.add_style("intro", "table-column", (
+            TableColumnProperties(columnwidth="150pt", breakbefore="auto"),
+        ))
+
+        doc.add_style("response", "table-column", (
+            TableColumnProperties(columnwidth="300pt", breakbefore="auto"),
+        ))
+
+        doc.add_style("row", "table-row", (
+            TableRowProperties(useoptimalrowheight="true", breakbefore="auto"),
+        ))
+
+        sheet = doc.sheet("Supplier evidence")
+
+        content = content_loader.get_manifest(
+            brief['frameworkSlug'],
+            'output_brief_response'
+        ).filter({'lot': brief['lotSlug']})
+
+        section = content.get_section('view-ods')
+
+        # two intro columns for boolean and dynamic lists
+        sheet.create_column(stylename="intro", defaultcellstylename="default")
+        sheet.create_column(stylename="intro", defaultcellstylename="default")
+
+        # HEADER
+        row = sheet.create_row("header")
+        row.write_cell(brief['title'], stylename="header",
+                       numbercolumnsspanned=str(len(responses) + 2))
+
+        # QUESTIONS
+        for question in section.questions:
+            if question.type in ('boolean_list', 'dynamic_list'):
```

Comment:
For the new flow and thus the ODS we are no longer asking any boolean list questions. (We currently look like we are from the current frameworks repo as the nice to have requirements question is a boolean list but this about to be replaced by a dynamic list question). Therefore I think we can lose the boolean list requirement.